### DISCUSSION TITLE
Profile picture issue

### DISCUSSION THREAD

**Discussion by 360WRACKER** [community/community#31480] [Other Feature Feedback, Questions, & Ideas]
Hello world!
 I do have problem that partly related to GitHub but since this place is very advanced maybe I can get the solution, here is the problem: some sites after login trough Google – upload my first ever used profile picture _which is very “cringy” and what i already successfully changed,_ to their sites, as GitHub for example.
 Even after deleting it everywhere in Google and changing it to another, sites still taking the picture from somewhere. I don't find any fast way to contact with Google (sounds funny) and the issue is not seems very popular to find answer on Q&A page, so possibly GitHub developers can help me to solve that little issue.
 The worry could be not that small, since every site that use Google login has access to some kind of hidden information about user.

**Comment by jfullstackdev** [comment]
you mean you uploaded the picture on GitHub and keeps showing on Google search ? 

try asking removing it here

https://support.github.com/contact/account

**Reply by 360WRACKER** [reply]
@jdevstatic I know It's hard to understand, I'm talking about registration on GitHub using Gmail address, after that somehow my main picture on GitHub was exactly my first pic that I was used for Google services in 2012-2013.
Looks like there's a library somewhere that sites use as GitHub for example. 
This is not the first time it's happen to me.
